frontend/packages/pipelines-plugin/src/types/pipeline.ts

```diff
@@ -67,6 +67,7 @@ export type PipelineTask = {
 
 export type PipelineWorkspace = {
   name: string;
+  optional?: boolean;
```

Comment:
Sneaky, best remove this as we are not currently (at this PR stage) supporting optional workspaces. We'll need another story to deal with that.

Reply:
Removed.